src/libraries/Microsoft.PowerFx.Core/Functions/IAsyncTexlFunction3.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    // A Texl function capable of async invokes, using IRContext. 
+    internal interface IAsyncTexlFunction3
```

Comment:
>IAsyncTexlFunction3 [](http://example.com/codeflow?start=23&length=19)

We really need to get rid of the IAsyncTexlFunction*... but that's outside the scope of this change